### DISCUSSION TITLE
Jan build error on windows 11

### DISCUSSION THREAD

**Discussion by letsrockroll** [janhq/jan#2222] [Get Help] [answered]
i installed  yarn --version (1.22.19)  node --version (v21.6.2)   make --version (GNU Make 4.4.1) 
but the "make dev" build failed quickly, can anyone help?  why does it has "EPERM" error?
really appreciate it! 

cd uikit && yarn config set network-timeout 300000 && yarn install && yarn build
yarn config v1.22.19
success Set "network-timeout" to "300000".
Done in 0.04s.
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
warning Pattern ["@janhq/core@link:./web/core"] is trying to unpack in the same destination "C:\\Users\\rick\\AppData\\Local\\Yarn\\Cache\\v6\\npm-@janhq-core-0.0.0\\node_modules\\@janhq\\core" as pattern ["@janhq/core@link:./electron/core"]. This could result in non-deterministic behavior, skipping.
warning Pattern ["@janhq/core@link:./server/core"] is trying to unpack in the same destination "C:\\Users\\rick\\AppData\\Local\\Yarn\\Cache\\v6\\npm-@janhq-core-0.0.0\\node_modules\\@janhq\\core" as pattern ["@janhq/core@link:./electron/core"]. This could result in non-deterministic behavior, skipping.
e**rror An unexpected error occurred: "EPERM: operation not permitted, unlink 'C:\\Users\\rick\\Documents\\ai\\jan\\node_modules\\.yarn-integrity'".**
info If you think this is a bug, please open a bug report with the information provided in "C:\\Users\\rick\\Documents\\ai\\jan\\uikit\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
make: *** [Makefile:10: build-uikit] Error 1

**Comment by letsrockroll** [comment]
i also got another error (EACCESS) after uninstalling the anti-virus software 

warning "workspace-aggregator-594c2828-927e-4257-8266-8d026274ad85 > jan-web > react-hot-toast > goober@2.1.14" has unmet peer dependency "csstype@^3.0.10".
error An unexpected error occurred: "EACCES: permission denied, lstat 'C:\\Users\\rick\\Documents\\ai\\jan\\node_modules\\@janhq\\core'".
info If you think this is a bug, please open a bug report with the information provided in "C:\\Users\\rick\\Documents\\ai\\jan\\uikit\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
make: *** [Makefile:10: build-uikit] Error 1

**Comment by louis-jan** [comment, marked as answer]
The issue has been addressed iirc, could you please help try again
